client/state/login/selectors.js

```diff
@@ -204,8 +198,7 @@ export const getAuthAccountType = ( state ) => {
  * @param  {object}   state  Global state tree
  * @returns {?boolean}         Error for the request.
  */
-export const isSocialAccountCreating = ( state ) =>
-	get( state, 'login.socialAccount.isCreating', null );
+export const isSocialAccountCreating = ( state ) => state.login.socialAccount?.isCreating ?? null;
```

Comment:
No need for optional chaining nor the fallback. `socialAccount` is always present and always has the `isCreating` property, which is always a boolean.

Reply:
Done.